Reply/app/src/main/java/com/materialstudies/reply/util/EmailUtils.kt

```diff
@@ -0,0 +1,17 @@
+package com.materialstudies.reply.util
+
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+
+object EmailUtils {
+
+    private const val DEFAULT_EMAIL_ID = 0L
+
+    private val _selectedEmailState = MutableStateFlow(DEFAULT_EMAIL_ID)
+    val selectedEmailState: StateFlow<Long> = _selectedEmailState.asStateFlow()
+
+    fun updateEmailId(emailId: Long) {
+        _selectedEmailState.value = emailId
+    }
+}
```

Comment:
Instead of creating this class, let's put this in EmailStore. Can we also update these to be 

`selectedEmailId`
`setSelectedEmailId(id: Long)`